src/components/Calendar/pageObject/index.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Calendar page object class.
+ * @class
+ * @tutorial calendar
+ */
+class PageCalendar {
+    /**
+     * Create a new PageCalendar page object.
+     * @constructor
+     * @param {string} rootElement - The selector of the PageCalendar root element.
+     */
+    constructor(rootElement) {
+        this.rootElement = rootElement;
+    }
+
+    /**
+     * Clicks the previous month button element.
+     * @method
+     */
+    clickPrevMonthButton() {
+        $(this.rootElement)
+            .$$('button[data-id=button-icon-element]')[0]
+            .click();
+    }
+
+    /**
+     * Clicks the next month button element.
+     * @method
+     */
+    clickNextMonthButton() {
+        $(this.rootElement)
+            .$$('button[data-id=button-icon-element]')[1]
+            .click();
+    }
+
+    /**
+     * Clicks the select year element.
+     * @method
+     */
+    clickSelectYear() {
+        $(this.rootElement)
+            .$('select#select-1')
```

Comment:
we can't use this since the id is generated
what happen if there are more selects in the same page?